### PR TITLE
Add periodic backups configuration commands

### LINKED ISSUE #454: It should be possible to configure db backup periodicity (enable/disable + hour)
https://github.com/Scalingo/go-scalingo/issues/118

### PR DESCRIPTION
**Pull request by EtienneM**
```
$ scalingo --app my-app --addon ad-d4903c5a-7b38-4fca-950a-54a34ae84ca4 backups-config --disable-scheduling
-----> Periodic backups are disabled

$ scalingo --app my-app --addon ad-d4903c5a-7b38-4fca-950a-54a34ae84ca4 backups-config --schedule-at 3
-----> Periodic backups will be done daily at 3:00 CEST
```

Fix #454

### CODE REVIEW

File: cmd/databases.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+)
+
+var (
+	databaseBackupsConfig = cli.Command{
+		Name:     "backups-config",
+		Category: "Addons",
+		Usage:    "Configure the periodic backups of a database",
+		Flags: []cli.Flag{appFlag, addonFlag, cli.IntFlag{
+			Name:  "schedule-at",
+			Usage: "Enable daily backups and schedule them at the specified hour of the day (UTC)",
```
Comment (Soulou) [inline cmd/databases.go]: Shouldn't we use the local timezone?

Comment (EtienneM) [reply cmd/databases.go]: Yes it's probably better UX

**Comment by Soulou** [comment]
And if no flag is used, shoudn't we display the current backups configuration?

**Comment by yannski** [comment]
For the record here is how Heroku does it https://devcenter.heroku.com/articles/heroku-postgres-backups#scheduling-backups

File: cmd/databases.go (hunk 1 above)
Comment (EtienneM) [reply cmd/databases.go]: Here your are:

```
$ scalingo --app my-app --addon ad-d4903c5a-7b38-4fca-950a-54a34ae84ca4 backups-config --scheduled-at 3
-----> Periodic backups will be done daily at 3:00 CEST
```

**Comment by Soulou** [comment]
@EtienneM what so you think, local time by default but ability to write "full tz" like heroku?

**Comment by EtienneM** [comment]
> @EtienneM what so you think, local time by default but ability to write "full tz" like heroku?

Yep I will do that

And I like that Heroku name the option `unschedule`. I will also update this command name.

**Pull request merged**